CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 * Changed Cargo Motor texture
 * Ancient Pedestals can now be found under "Magical Gadgets"
 * Changed item order in guide for the Villager Rune and Nether Goo (All runes are now grouped together)
+* Added enchantment level limit for both auto enchanter and auto disenchanter
```

Comment:
Why is that a "change" though and not an "addition"... It literally starts with "Added ..."